server/models/plugins/pluginList.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
```

Comment:
Is a model required? Could we use a simple array initialized in index.js?

Reply:
yes, to keep track of the List of available plugin, i understand there have to be a central defination

Reply:
I understand I only meant about the method in which we store the plugin list